src/reason_pprint_ast.ml

```diff
@@ -5130,19 +5125,24 @@ class printer  ()= object(self:'self)
       | Pcf_method (s, pf, Cfk_virtual ct) ->
         let opening = match pf with
           | Private ->
-            let privateVirtualName = [atom "private"; atom "virtual"; atom s.txt] in
+            let privateVirtualName = [atom "virtual"; atom s.txt] in
             let openingTokens =
               (makeList ~postSpace:true ~inline:(false, true) ~break:IfNeed privateVirtualName) in
-            label ~space:true (atom "method") openingTokens
+            label ~space:true (atom "pri") openingTokens
           | Public ->
             let virtualName = [atom "virtual"; atom s.txt] in
             let openingTokens =
               (makeList ~postSpace:true ~inline:(false, true) ~break:IfNeed virtualName) in
-            label ~space:true (atom "method") openingTokens
+            label ~space:true (atom "pub") openingTokens
         in
         formatTypeConstraint opening (self#core_type ct)
       | Pcf_method (s, pf, Cfk_concrete (ovf, e)) ->
-        let methodText = if ovf == Override then "method!" else "method" in
+        let methodText = if ovf == Override then "method!" else
```

Comment:
I think this is a bug! I believe we want to print `pri!` or `pub!`. There may be no test case that caught this? Could you at least add a small test case to catch this too?